### PR TITLE
Brewfile

### PR DESCRIPTION
**Pull request by rye**
Add a Brewfile to the repository.

This will assist with onboarding of new developers who have macOS systems.

(This _might_ (??????) be cross-compatible with LinuxBrew, which would be awesome—`homebrew-bundle` claims that it might work but is not officially supported.)

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+brew "git-lfs"
```
Comment (hawkrives) [inline Brewfile]: We don't use this anywhere, and AFAICR we rewrote history to completely remove it from our repo, so we can kill it off in the Brewfile.

**Review by hawkrives — DISMISSED** [review]

**Comment by hawkrives** [comment]
(Also, uh… how do you use this?)

**Comment by rye** [comment]
`brew bundle [install]` will:
>Install or upgrade all dependencies in a Brewfile.

- - -

Docs:

```
brew bundle command:
    Bundler for non-Ruby dependencies from Homebrew.

    bundle [install] [-v|--verbose] [--no-upgrade] [--file=path|--global]:
    Install or upgrade all dependencies in a Brewfile.

    brew bundle dump [--force] [--describe] [--file=path|--global]
    Write all installed casks/formulae/taps into a Brewfile.

    brew bundle cleanup [--force] [--zap] [--file=path|--global]
    Uninstall all dependencies not listed in a Brewfile.

    brew bundle check [--no-upgrade] [--file=path|--global] [--verbose]
    Check if all dependencies are installed in a Brewfile. Missing dependencies are listed in verbose mode. check will exit on the first category missing a dependency unless in verbose mode.

    brew bundle exec command
    Run an external command in an isolated build environment.

    brew bundle list [--all|--brews|--casks|--taps|--mas] [--file=path|--global]
    List all dependencies present in a Brewfile, optionally limiting by types.
    By default, only brew dependencies are output.

    If -v or --verbose are passed, print verbose output.

    If --no-upgrade is passed, don't run brew upgrade outdated dependencies.
    Note they may still be upgraded by brew install if needed.

    If --force is passed, uninstall dependencies or overwrite an existing Brewfile.

    If --file=path is passed, the Brewfile path is set accordingly
    Use --file=- to output to console.

    If --global is passed, set Brewfile path to $HOME/.Brewfile.

    If --describe is passed, output a description comment above each line.
    This comment will not be output if the dependency does not have a description.

    If -h or --help are passed, print this help message and exit.
```

**Review by hawkrives — DISMISSED** [review]
I'd also be OK with pulling the applesimutils out into another Brewfile that you could run if you wanted to do testing, like something that could run as part of the testing/e2e init process.

**Review by hannesmcman — DISMISSED** [review]
Can we add the instructions for using this to the README in this PR?

**Comment by rye** [comment]
I would complain about out-of-scope but that makes sense.

**Review by hannesmcman — APPROVED** [review]
thx 😈

**Pull request merged**